lucille-core/src/main/java/com/kmwllc/lucille/indexer/OpenSearchIndexer.java

```diff
@@ -92,46 +96,60 @@ protected void sendToIndex(List<Document> documents) throws Exception {
       return;
     }
 
-    BulkRequest bulkRequest = new BulkRequest(index);
+    BulkRequest.Builder br = new BulkRequest.Builder();
 
-    // determine what field to use as id field and iterate over the documents
     for (Document doc : documents) {
-      Map<String, Object> indexerDoc = getIndexerDoc(doc);
+      Map<String, Object> indexerDoc = doc.asMap();
 
       // remove children documents field from indexer doc (processed from doc by addChildren method call below)
       indexerDoc.remove(Document.CHILDREN_FIELD);
```

Comment:
i know this PR didn't change this line, but I'm wondering: why is this being done separately from the addChildren call below? i.e. why aren't those two lines grouped?

Reply:
Ticket was created